vignettes/Introduction_To_BiocParallel.Rmd

```diff
@@ -0,0 +1,961 @@
+---
+title: "**Introduction to *BiocParallel***"
+author: "Phyllis Atieno"
+date: "Compiled: 18 October, 2022"
+package: BiocParallel
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document
+  
+---
+
+Introduction
+============
+
+Numerous approaches are available for parallel computing in R. The CRAN Task View for
+high performance and parallel computing provides useful high-level summaries and [package
+categorization.](http://cran.r-project.org/web/views/HighPerformanceComputing.html)
+Most Task View packages cite or identify one or more of **snow** , **Rmpi**, **multicore** or **foreach** as
```

Comment:
In the original these packages were italicized and linked to their CRAN 'landing pages', `[*snow*](https://cran.r-project.org/package=snow)`